script/check_docker_compose.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
@axw , it seems this script was not backported to the 7.12 branch, do we need it?

Reply:
Sorry, I didn't see your question earlier. I'm not sure if we need the automation at all in 7.12, since 7.13 won't be too far away. WDYT about closing this and just backporting to 7.x and 7.13?